src/crx.js

```diff
@@ -60,9 +61,7 @@ class ChromeExtension {
         return crx3(selfie.privateKey, publicKey, contents);
       }
 
-      var signature = selfie.generateSignature(contents);
-
-      return selfie.generatePackage(signature, publicKey, contents);
+      return crx2(selfie.privateKey, publicKey, contents);
```

Comment:
Since crx3 is now default, maybe it would be better to switch this and with `if'ed` crx3?
I mean, something along the lines of:
```js
if (selfie.version === 2) {
  return crx2(...);
}

return crx3(...);
```